src/bower_config.ts

```diff
@@ -24,10 +24,9 @@ function bowerConfigContents(root?: string): string {
   let contents: string;
 
   try {
-    contents = fs.readFileSync(bowerConfigPath(root)).toString();
+    contents = fs.readFileSync(bowerConfigPath(root), 'utf-8');
```

Comment:
@TimvdLippe and I just merged a PR that makes a similar improvement, you may want to rebase/merge to pull in those changes